tests/unit/bokeh/plotting/test_contour.py

```diff
@@ -27,6 +30,12 @@
     from_contour,
 )
 
+if TYPE_CHECKING:
+    from typing_extensions import TypeAlias
+
+    # This is to make the fixture typing simpler
+    XYZ_Levels: TypeAlias = tuple[list[float], list[float], list[list[float]], list[float]]
```

Comment:
This is better than no type at all, but still not conveying enough context (unless we mentally parse the name) and quite busy. In TypeScript I would usually use named fields in tuple types:
```ts
type XYZ_Levels = [X: number[], Y: number[], Z: number[][], Levels: number[]]
```
Sadly Python's typing doesn't support such construct, so perhaps let's use this:
```suggestion
    X = list[float]
    Y = list[float]
    Z = list[list[float]]
    Levels = list[float]
    XYZ_Levels: TypeAlias = tuple[X, Y, Z, Levels]
```

Reply:
I almost prefer no typing at all for `XYZ_Levels`, but I know that isn't acceptable! For nearly all of the tests here the `x`, `y`, `z` and `levels` are dummy data (but valid) to allow testing of visual properties, so the more we write for these types the more emphasis we are giving to them rather than the important stuff.

As @mattpap has approved this PR as it is, I propose to push one small change to add another comment to help with this, then I'll leave it for 24 hours in case anyone has any more comments before I self-merge.